.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   extends: [require.resolve('./lib/configs/recommended'), 'plugin:eslint-plugin/all'],
   plugins: ['eslint-plugin'],
   rules: {
+    'import/extensions': 'off',
```

Comment:
In order for the JS files to be loaded into `.mjs`, we need an extension.

This library seems to be using a mix of module JS and common JS now, so we'll want to turn this rule off.